am2opac.py

```diff
@@ -474,6 +477,85 @@ def process_last_issue(issn):
         logger.info(u"Impossível recuperar o último fascículo para o períodico: %s" % journal.title)
 
 
+def process_articles_ssm(issn):
+
+    connect(**config.MONGODB_SETTINGS)
+
+    journal = models.Journal.objects.get(scielo_issn=issn)
+
+    logger.info(u"Enviando os artigos do periódico: {0} para SSM".format(journal.title))
+
+    for article in models.Article.objects.filter(journal=journal):
+
+        article.htmls = []
+        article.save()
+
+        try:
+            if article.xml:
+                # O caminho deve ser até o acrônimo do periódico.
+                filename = os.path.join(config.ARTICLE_FILE_PATH, article.xml)
+                try:
+                    fp = open(filename)
+                except IOError:
+                    logger.error(u"Erro ao tentar coletar o arquivo {0}".format(filename))
+                    continue
+
+                list_dict_html = []
+                xml_etree = etree.parse(StringIO(fp.read()))
+
+                client_ssm = client.Client()
+
+                for lang, output in packtools.HTMLGenerator.parse(xml_etree, valid_only=False):
+                    dict_htmls = {}
+                    source = etree.tostring(output, encoding="utf-8",
+                                                    method="html",
+                                                    doctype=u"<!DOCTYPE html>")
+
+                    dict_htmls['lang'] = lang
+
+                    filename = os.path.basename(getattr(fp, 'name', None)).split('.')
```

Comment:
aqui se o nome do arquivo ou no caminho absoulto do arquivo houver um ponto, pode dar problema.
a forma mais segura é usar o [splitext](https://docs.python.org/2/library/os.path.html#os.path.splitext) 
tem um exemplo no [StackOverflow](https://stackoverflow.com/questions/541390/extracting-extension-from-filename-in-python#541394)